### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by ejohnstown**
1. Changed "ssh_client" to "ssh". This example is intended to use the usual ssh client.
2. Changed a later section header to be different from an earlier one. (examples)
3. Added some backticks around some short phrases that are meant to be commands or something included in code.
4. Added some links.
5. Update the build directions.
6. A bunch of lines were longer than 80 columns or got really close. These were rewrapped to be a little shorter. Whitespace only.
7. Remove non-markdown README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,10 @@ wolfSSL's Embeddable SSH Server
 dependencies
 ------------
 
-[wolfSSH](https://www.wolfssl.com/wolfssh/) is dependent on [wolfCrypt](https://www.wolfssl.com/download/).
-The simplest configuration of wolfSSL required for wolfSSH is the default build.
+[wolfSSH](https://www.wolfssl.com/wolfssh/) is dependent on
+[wolfCrypt](https://www.wolfssl.com/download/).
```
Comment (anhu) [inline README.md:11]: this is somewhat confusing as wolfcrypt isn't a package you can download.  You might want to say "which is part of wolfSSL" or just replace wolfCrypt with wolfSSL.

File: README.md [hunk 2]
```diff
@@ -35,40 +39,41 @@ From the wolfSSH source directory run:
...
-to connect to. From the terminal run:
+The directory `examples` contains an echoserver that any client should
+be able to connect to. From the terminal run:
 
     $ ./examples/echoserver/echoserver -f
 
 The option `-f` enables echo-only mode. From another terminal run:
 
-    $ ssh_client jill@localhost -p 22222
+    $ ssh jill@localhost -p 22222
```
Comment (anhu) [inline README.md:66]: optional: provide a link to the ssh homepage so reader knows where to download.

**Review by anhu — COMMENTED** [review]
Excellent cleanup!  Just two minor points.

File: README.md (hunk 2 above)
Comment (ejohnstown) [reply README.md:66]: It is typically already on everybody's computer. Except for Windows people, I guess.

File: README.md (hunk 1 above)
Comment (ejohnstown) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (anhu) [reply README.md:66]: True enough.

**Review by anhu — APPROVED** [review]
Looks good.  :)

**Pull request merged**